storj/http.py

```diff
@@ -41,7 +40,7 @@ class Client(object):
         public_key_hex ():
     """
 
-    logger = logging.getLogger('%s.Client' % __name__)
+    logger = logging.getLogger(web_socket.Client.__name__)
```

Comment:
I need to learn more about the logger. I changed this line when I changed the import statement from `from .web_socket import Client` to `from storj import web_socket`. I was basically applying blanket changes to everywhere I saw Client.

I'll make the change suggested.